client/components/TestPlanUpdater/TestPlanUpdaterModal.css

```diff
@@ -0,0 +1,80 @@
+.cancel-button {
+    width: 100%;
+    justify-self: end;
+}
+
+.completion-alert {
+    margin-bottom: 0em;
+}
+
+.backup-checkbox {
+    grid-column: 1/2;
+}
+
+.loading-spinner {
+    align-self: center;
+    margin: 40px 0;
+}
+
+.loading-spinner-modal {
+    height: 250px;
+}
+
+.modal-50w {
+    max-width: 100%;
+    width: 50%;
+}
+
+.new-version {
+    grid-column: 1/3;
+    margin-bottom: 1em;
+}
+
+.side-button-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 0.5em;
+}
+
+.side-button-container.no-backup {
+    grid-column: 2/3;
+}
+
+.submit-button {
+    width: 100%;
+    justify-self: end;
+}
+
+.submit-buttons-row {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 100%;
+}
```

Comment:
I noticed the button text was wrapping, and I think it should be possible to keep the button text on one line. I think the css can be simplified slightly! And the HTML can be simplified too - you can remove the divs with the classes which are no longer needed.

```css
.side-button-container {
    display: grid;
    grid-template-columns: auto auto;
    grid-gap: 0.5em;
}
/* Can be removed */
/*
.side-button-container.no-backup {
    grid-column: 2/3;
}

.submit-button {
    width: 100%;
    justify-self: end;
}

.submit-buttons-row {
    display: grid;
    grid-template-columns: 1fr 1fr;
    width: 100%;
}
*/
```

Reply:
<img width="726" alt="Screen Shot 2022-03-22 at 3 10 33 PM" src="https://user-images.githubusercontent.com/2789098/159557584-441d412c-d1d2-48c6-91b5-fe90952e6fe4.png">

The backup checkbox gets rendered conditionally, so I need the styles. to keep the checkbox on the left and the two buttons on the right.

Reply:
Ah, I get it, there's some styles and HTML to keep the checkbox above the buttons. Ok, so the main thing I wanted to address was the fact the button didn't have enough space and was wrapping. I would say, if you could figure out a way to fix that, that's the main thing motivating my suggestion. I'm happy to look at it together if that helps.

Reply:
@alflennik I've addressed the text wrapping. It's on sandbox. Mind taking a look?

Reply:
Looks great.